src/lang/en.json

```diff
@@ -882,5 +882,6 @@
     "useRemoteBrowser": "Use a Remote Browser",
     "deleteRemoteBrowserMessage": "Are you sure want to delete this Remote Browser for all monitors?",
     "GrafanaOncallUrl": "Grafana Oncall URL",
-    "Browser Screenshot": "Browser Screenshot"
+    "Browser Screenshot": "Browser Screenshot",
+    "wayToWriteWhapiPhoneNumber": "The phone number with the international prefix, but without the plus sign at the start"
```

Comment:
The regex also allows `-`. 

Is this intentional? Does the helptext need adjusting?

Reply:
Actually the API doesn't work with the `-` so I removed it